topics/materials-science/tutorials/muon-stopping-sites-muairss-uep/tutorial.md

```diff
@@ -0,0 +1,220 @@
+zenodo_link: https://sandbox.zenodo.org/record/1031546
+questions:
+- "What are stopping sites and why is finding them important?"
+- "What computational techniques can be used to find stopping sites?"
+- "What materials can I apply these techniques to?"
+- "How can Galaxy help me with my analysis?"
+objectives:
+- "Understand why finding stopping sites is useful"
+- "Apply the AIRSS and UEP methods together to find stopping sites"
+- "Understand the limitations of these tools and techniques"
+time_estimation: 1H
+key_points:
+- Galaxy supports multiple PyMuonSuite tools which can be used together to find muon stopping sites
+- The simulations require the results of DFT calculations using CASTEP as a starting point
+- The UEP approach can be performed entirely via Galaxy, or CASTEP can be used externally
+contributors:
+- elichad
+- leandro-liborio
+- patrick-austin
+subtopic: muon-spectroscopy
+priority: 2
+abbreviations:
+  AIRSS: Ab Initio Random Structure Search
+  DFT: Density Functional Theory
+  UEP: Unperturbed Electrostatic Potential
+---
+
+
+# Introduction
+
+The problem of identifying the stopping site of a muon is often key in the
+interpretation of experimental results, and also one of those that computational
+tools are most commonly used to solve. Roughly, one can imagine that at the
+atomic level, muon spectroscopy experiments go something like this:
+
+- the polarized muon beam is fired at the sample, and the muons travel a certain
+  depth inside the target;
+- through inelastic collisions with the atoms of the lattice, muons shed their
+  kinetic energy, which turns into heat or causes small amounts of damage to the
+  crystal, until it becomes low enough that it is comparable to their potential
+  energy in the electrostatic fields generated by electrons and nuclei alike;
+- at that point, Coulomb forces start driving the muon, and eventually, it loses
+  as much energy as possible and comes to a stop in a position that represents a
+  global, or at least local, minimum for its electrostatic potential energy,
+  where it spends the rest of its time until it decays.
+
+![Graphical illustration of muons being implanted in a sample, then decaying to a positron and being detected](../../images/muon_spectroscopy_diagram.png)
+
+Where these minima of the potential actually are isn't always obvious. In some
+cases, it's trivial - for example in organic molecules the mechanism is often
+the same as for regular hydrogenation, and so the muon, having captured an
+electron and formed a muonium pseudo-atom, ends up breaking a double or triple
+bond and attaching itself to the molecule by using one of the unpaired
+electrons. However in other crystalline structures it can be a lot harder. A
+reliable atomistic simulation that accurately predicted the potential through
+which the atoms interact, though, can identify these possible sites. In order to
+find them, however, one generally needs to use a brute force approach to explore
```

Comment:
This doesn't quite make sense - if the atomistic simulation finds the sites, why do we need to find them with brute force?

I _think_ this means that only some of the possible sites will actually be occupied by muons, hence this suggestion

```suggestion
find which possible sites are actual muon stopping sites, however, one generally needs to use a brute force approach to explore
```

Reply:
Yeah I copied this bit from the [PyMuonSuite GH wiki](https://github.com/muon-spectroscopy-computational-project/pymuon-suite/wiki/MuAIRSS) so am also unclear what exactly it means.

Having read the intro to [Computational prediction of muon stopping sites using ab initio random structure searching (AIRSS)](https://aip.scitation.org/doi/pdf/10.1063/1.5024450) that seems to imply that just using UEP analysis "is not fully reliable" as:
>In the case of muonium, it has been suggested that the screening
provided by the electron makes the muon less sensitive to the
electrostatic potential of the host.6,8,9 With regards to charged
muons, it has been proposed that the inability to account for
the muon-host interactions is what decreases the efficiency
of the UEP method in predicting the muon stopping sites.

So my reading would be:
 - We can identify locations that are likely in some organic molecules  because we can assume (similar to hydrogen) we'll latch onto (e.g.) double bonds.
 - But that in general (e.g. in "other crystalline structures") you could use UEP ("reliable atomistic simulation"?) alone to find potential sites.
 - **But** since that doesn't **always** work, you might want to brute force it using (for example) AIRSS.

I'm considering rewritting this whole paragraph to resemble the points in the paper slightly more:
```
Where these minima of the potential actually are isn't always obvious. In some
cases it's trivial, for example in organic molecules the mechanism is often the
same as for regular hydrogenation. The muon, having captured an electron and
formed a muonium pseudo-atom, ends up breaking a double or triple bond and
attaching itself to the molecule by using one of the unpaired electrons. However
in other, crystalline structures it can be a lot harder. While an atomistic
simulation that accurately predicts the potential through which the atoms
interact can be used identify possible sites, this approach is not fully
reliable (for example when applied to fluorides) {% cite Liborio2018 %}. In
order to avoid such limitations, one can use a brute force approach to explore
all the possible positions in which the muon may end up. {AIRSS} is one such
method {% cite Liborio2018 %}.
```

Whatever we decide on here, and wherever other changes are made to content which I've just copied wholesale from an existing source, we should probably make an effort to update the source? Joel's already been doing this for some changes for the MSS tutorial.

Reply:
That new paragraph is good, and yes I'd say update the wiki with that too